truss/templates/server/requirements.txt

```diff
@@ -9,6 +9,7 @@ python-json-logger==2.0.2
 pyyaml==6.0.0
 fastapi==0.95.0
 uvicorn==0.21.1
+uvloop==0.17.0
```

Comment:
should we explcitly add this to pyproject.toml too?

Reply:
Through pyproject.toml uvicorn automatically pulls in uvloop, I'm not sure we need it there explicitly.